app/models/enhanced_loan_data_export.rb

```diff
@@ -74,9 +50,10 @@ def questions
   end
 
   def question_sets
-    # We want self to come first for deterministic behavior in specs. After that it doesn't really matter.
-    # self_and_descendants orders by depth so we are good.
-    division.self_and_descendants.flat_map { |d| QuestionSet.where(division: d).order(:kind).to_a }
```

Comment:
the previous version of this meant that if one is in a division that inherits question set(s) from ancestor divisions when making a data export, those inherited question sets were skipped.